dvc/command/dag.py

```diff
@@ -53,6 +52,8 @@ def _collect_targets(repo, target, outs):
 def _transform(repo, outs):
     import networkx as nx
 
+    from dvc.stage import Stage
```

Comment:
`dvc.stage` imports a lot of things (eg: all deps and output which then import trees, etc.). So, lazily importing here.